test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -559,6 +559,19 @@ firrtl.circuit "ExternalModule" {
   }
 }
 
+// -----
+firrtl.circuit "bundleop" {
+  firrtl.module @bundleop(in %x : !firrtl.bundle<a: flip<uint<8>>>, in %y : !firrtl.uint<8>) {
+    // check: %w_a_a = firrtl.wire  : !firrtl.uint<8>
+    // check: %w_b = firrtl.wire  : !firrtl.uint<8>
+    // check: firrtl.connect %x_a, %w_a_a : !firrtl.uint<8>, !firrtl.uint<8>
+    // check: firrtl.connect %w_b, %y : !firrtl.uint<8>, !firrtl.uint<8>
+    %w = firrtl.wire : !firrtl.bundle<a: bundle<a: flip<uint<8>>>, b: uint<8>>
+    %0 = firrtl.bundle %x, %y : (!firrtl.bundle<a: flip<uint<8>>>, !firrtl.uint<8>) -> !firrtl.bundle<a: bundle<a: flip<uint<8>>>, b: uint<8>>
+    firrtl.connect %w, %0 : !firrtl.bundle<a: bundle<a: flip<uint<8>>>, b: uint<8>>, !firrtl.bundle<a: bundle<a: flip<uint<8>>>, b: uint<8>>
```

Comment:
This seems a bit weird to me as it's letting you get a reference to `x.a` and drive that. 

Maybe this is what we want, though.

Reply:
Unrelated to your comment, it looks like I accidentally lower cased the entire paragraph